src/index.js

```diff
@@ -68,16 +68,6 @@ function length (history) {
 }
 // /length
 
-// isHistory: check for a valid history object
-function isHistory (history) {
-  return typeof history.present !== 'undefined' &&
-    typeof history.future !== 'undefined' &&
-    typeof history.past !== 'undefined' &&
-    Array.isArray(history.future) &&
-    Array.isArray(history.past)
-}
-// /isHistory
-
```

Comment:
Needed this for testing as well.. exported at the end of file.. dunno of this is the right approach, please correct me if I'm wrong.